OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -290,6 +291,17 @@ public interface IProvidesAssetBrowserColorPickerPalettes
 		IEnumerable<string> ColorPickerPaletteNames { get; }
 	}
 
+	public interface IColorPickerManagerInfo : ITraitInfoInterface
+	{
+		Color ColorPickerPaletteColor { get; set; }
+		(float hMin, float hMax, float sMin, float sMax, float vMin, float vMax) GetHSVRange { get; }
+		string GetPreviewActor(string faction = null);
```

Comment:
Displaying an actor is a mod-specific implementation detail (a mod might want to recolor a static png image of a base, for example) so i'm not convinced that `GetPreviewActor` belongs on this interface.

Reply:
I'm not sure how to abstract it then. The definition needs to de transferred to another trait / widget.

Reply:
perhaps add a `ActorColorPickerPreviewWidget : ActorPreviewWidget`